### PR TITLE
Use wrapped function api and add inference benchmark

### PR DESCRIPTION
**Pull request by juncaipeng**
Use wrapped function api and add inference benchmark

### CODE REVIEW

**Review by michaelowenliu — DISMISSED** [review]

File: paddleseg/models/layers/wrap_functions.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+# limitations under the License.
+
+import paddle
+import paddle.nn as nn
+"""
+Warp the functon api, so the normal and quantization training can use the same network.
+"""
+
+
+class add(nn.Layer):
```
Comment (michaelowenliu) [inline paddleseg/models/layers/wrap_functions.py]: class Add

File: paddleseg/models/layers/wrap_functions.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, x, y, name=None):
+        return paddle.add(x, y, name)
+
+
+class subtract(nn.Layer):
```
Comment (michaelowenliu) [inline paddleseg/models/layers/wrap_functions.py]: Class Subtract

File: paddleseg/models/layers/wrap_functions.py (hunk 1 above)
Comment (juncaipeng) [reply paddleseg/models/layers/wrap_functions.py]: Done

File: paddleseg/models/layers/wrap_functions.py (hunk 2 above)
Comment (juncaipeng) [reply paddleseg/models/layers/wrap_functions.py]: Done

**Review by michaelowenliu — APPROVED** [review]

**Pull request merged**